packages/cli/package.json

```diff
@@ -29,7 +29,7 @@
     "lint": "yarn run --top-level eslint -c .eslintrc.js ",
     "prepublish": "",
     "prepack": "yarn run build && oclif manifest && oclif readme",
-    "test": "TZ=UTC yarn jest --runInBand"
+    "test": "TZ=UTC yarn jest --runInBand --forceExit --silent"
```

Comment:
Out of curiosity, what do `--forceExit` and `--silent` do here? 

As in, why is this needed/what is the intended behaviour?

Reply:
*silent* removes noise from output so tests can be seen rather than lots of logs warning and errors from testing failure states. 

*forceExit* because there seem to be some tests that dont stop in time. leszek and i saw this while working on another pr that didnt get merged in yet. conractkit tests have had this flag for a while already.